Common/include/interpolation_structure.hpp

```diff
@@ -253,5 +285,98 @@ class CMirror : public CInterpolator {
    * \param[in] config - Definition of the particular problem.
    */
   void Set_TransferCoeff(CConfig **config);
+  
+};
+
+/*!
+ * \brief Sliding mesh approach
```

Comment:
Again here, for the name CSlidingMesh would a name focused on the interpolation type be more appropriate than "SlidingMesh"? This method seems to be more a specific type of interpolation.